v3.1/glfw/util.go

```diff
@@ -1,11 +1,53 @@
 package glfw
 
+//#include <stdlib.h>
 //#include "glfw/include/GLFW/glfw3.h"
 import "C"
 
+import (
+	"reflect"
+	"unsafe"
+)
+
 func glfwbool(b C.int) bool {
 	if b == C.GL_TRUE {
 		return true
 	}
 	return false
 }
+
+// Strs takes a list of Go strings (with or without null-termination) and
```

Comment:
Since it's no longer exported, it's called `strs`, not `Strs`. The documentation should be updated too.